x-pack/plugins/ml/public/application/components/anomaly_results_view_selector/anomaly_results_view_selector.test.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { Router } from 'react-router-dom';
+import { render, fireEvent } from '@testing-library/react';
+import { createBrowserHistory } from 'history';
+
+import { I18nProvider } from '@kbn/i18n/react';
+
+import { AnomalyResultsViewSelector } from './index';
+
+describe('AnomalyResultsViewSelector', () => {
+  test('creates selector with correctly selected value', () => {
```

Comment:
could you please update test suites ids to follow the same naming convention as in `x-pack/plugins/ml/public/application/settings/settings.test.tsx`, i.e. start with `should ...`

Reply:
Done in https://github.com/elastic/kibana/pull/68663/commits/e67c1fb40fd8180d877148fc18d379610b21e5d6